contributing/changelog-process.md

```diff
@@ -1,9 +1,9 @@
 # Changelog Process
 
+- [PR Changelog check](#pr-changelog-check)
 - [Changelog format](#changelog-format)
 - [Changelog entry guidelines](#changelog-entry-guidelines)
 - [Script for creating changelog entry files](#script-for-creating-changelog-entry-files)
-- [PR Changelog check](#pr-changelog-check)
 - [Unreleased section of CHANGELOG.md automatic update](#unreleased-section-of-CHANGELOG.md-automatic-update)
```

Comment:
I believe the section you want to move to the top is `Script for creating changelog entry files` right?

Reply:
I am sorry :/ yes let me fix that